README.md

```diff
@@ -2,4 +2,112 @@
 
 English | [简体中文](README.zh-CN.md)
 
-This is a template repository for customising cactbot with user directory.
\ No newline at end of file
+This project is a template project for creating Cactbot customized triggers/timelines or custom user configurations, extending the scope and possibilities of customization based on the functionality of Cactbot's user folder override.
+
+**Hint** Since this project uses webpack to package all configuration files as *.js files with the corresponding module name, it can work in ACT even with the versions like CafeACT which do not support subfolders.
```

Comment:
```suggestion
```

I think it is better to remove this section? cuz it is only available to CN players.